Source/WebCore/style/StyleAdjuster.cpp

```diff
@@ -968,6 +968,14 @@ void Adjuster::adjustForSiteSpecificQuirks(RenderStyle& style) const
             style.setUserSelect(UserSelect::None);
     }
 
+#if PLATFORM(IOS_FAMILY)
+    if (m_document->quirks().needsGoogleMapsScrollingQuirk()) {
+        static MainThreadNeverDestroyed<const AtomString> className("PUtLdf"_s);
```

Comment:
Yeah... If I understand correctly, it would be better to check for `div .UJrD4b` which should always be there on the page and what I initially set to do, but I was having trouble accessing the `<body>` that we needed to change the style on, which is why I choose to search for this class name since it's particular to the print page.